data/functions.sql

```diff
@@ -41,12 +41,15 @@ CREATE OR REPLACE FUNCTION mz_calculate_poi_level(
     railway_val text,
     shop_val text,
     tourism_val text,
-    waterway_val text
+    waterway_val text,
+    way_area real
 )
 RETURNS SMALLINT AS $$
+DECLARE
+  zoom smallint;
 BEGIN
-    RETURN (
-        CASE WHEN aeroway_val IN ('aerodrome', 'airport') THEN 9
+  zoom =
+        CASE WHEN aeroway_val IN ('aerodrome', 'airport') THEN 13
```

Comment:
Yup. Airports can be quite large features and having them pop up at zoom < 9 didn't seem right. I suspect we'll have to revisit these levels anyway in a future task.